codegen-winter/src/air/boundary_constraints.rs

```diff
@@ -21,23 +21,46 @@ pub(super) fn add_fn_get_assertions(impl_ref: &mut Impl, ir: &AirIR) {
         let assertion = format!(
             "result.push(Assertion::single({}, 0, {}));",
             col_idx,
-            constraint.to_string()
+            constraint.to_string(false)
+        );
+        get_assertions.line(assertion);
+    }
+
+    // add the constraints for the auxiliary columns for the first boundary.
+    for (col_idx, constraint) in ir.aux_first_boundary_constraints() {
+        let assertion = format!(
+            "result.push(Assertion::single({}, 0, {}));",
+            col_idx,
+            constraint.to_string(true)
```

Comment:
These ones and the aux_last_boundary_constraints should both be added to a `get_aux_assertions` function instead, so there should be a new function `add_fn_get_aux_assertions` for handling that

Reply:
Thanks, fixed.